node/overseer/overseer-gen/README.md

```diff
@@ -11,17 +11,18 @@ declarative.
 ```rust
     #[overlord(signal=SigSigSig, event=Event, gen=AllMessages, error=OverseerError)]
     pub struct Overseer {
-        #[subsystem(MsgA)]
+        #[subsystem(MsgA, sends: [MsgB])]
         sub_a: AwesomeSubSysA,
 
-        #[subsystem(MsgB)]
+        #[subsystem(MsgB, sends: [MsgA])]
         sub_b: AwesomeSubSysB,
     }
 ```
 
 * Each subsystem is annotated with `#[subsystem(_)]` where `MsgA` respectively `MsgB` are the messages
 being consumed by that particular subsystem. Each of those subsystems is required to implement the subsystem
-trait.
+trait with the correct trait bounds. Commonly this is achieved
+by using `#[subsystem]` and `#[contextbounds]` macro.
```

Comment:
nit: Maybe a short explanation about `#[contextbounds]` usage will be useful here.

Reply:
Done